packages/react-components/react-tooltip/stories/Tooltip/TooltipIcon.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Label, makeStyles, mergeClasses, tokens, Tooltip, useId } from '@fluentui/react-components';
+import { Info16Regular } from '@fluentui/react-icons';
+import type { TooltipProps } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    columnGap: tokens.spacingVerticalS,
+  },
+  visible: {
+    color: tokens.colorNeutralForeground2BrandSelected,
+  },
+});
+
+export const Icon = (props: Partial<TooltipProps>) => {
+  const styles = useStyles();
+  const iconId = useId('icon');
+  const contentId = useId('content');
+  const [visible, setVisible] = React.useState(false);
+
+  return (
+    <div aria-owns={visible ? contentId : undefined} className={styles.root}>
+      <Label>This is an icon with a Tooltip to show extra information</Label>
+      <Tooltip
+        content={{
+          children: 'Content must never contain focusable elements.',
+          id: contentId,
+        }}
+        positioning="above-start"
+        withArrow
+        relationship="label"
+        onVisibleChange={(e, { visible }) => setVisible(visible)}
```

Comment:
The `visible` parameter here will clash with your definition on top

```suggestion
        onVisibleChange={(e, data) => setVisible(data.visible)}
```